plaintext/2023/08/tenpo-li-lili.md

```diff
@@ -6,12 +6,24 @@ authors:
 proofreaders: null
 date: 2023-08-15
 date-precision: day
-tags: null
+tags:
+  - original
+  - poetry
+  - competition winner
+
+  # covers:
+  - cover
+  - music
+  - vocal synthizer
```

Comment:
synthizer :sob:
my concern about tagging extratextual things is even more intense here, given the only official version is text